src/portfolio-rebuild/index.html

```diff
@@ -0,0 +1,60 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Document</title>
+</head>
+<body>
+<nav>
+  <ul class="topnav">
+    <li><a class="active" href="#About">About</a></li>
+    <li><a href="#Portfolio">Portfolio</a></li>
+    <li><a href="#contact">Contact</a></li>
+    </ul>
+</nav>
+
+<div class="maincontainer">
+<div><header>RYAN BLIZZARD</header>
```

Comment:
this div right here needs flexbox to center the inner items. in your css for that div : 
`  display: flex;`
   ` flex-direction: column;`
    `justify-content: center;`
   ` align-items: center;`